Yoast/comment-hacks
# Preserve the active tab when saving settings

Uses the same implementation from Yoast SEO to preserve the active tab when saving settings and the admin page reloads.

To test:
- go in the settings page
- go in a tab other than the first one
- press "Save Changes"
- after the page reload, check the active tab is the one that was displayed before saving

Fixes #18
